### ISSUE TITLE
Start Listener is not listening for milli seconds

### ISSUE THREAD

**Issue by sudha53633** [brightcove/videojs-overlay#36]
I would like appreciate your efforts on making video overlayjs plugin. It's very easy to use. I'm very much thankful to that.

I facing an issue with startListener_ reading videojs player time, It's skipping around 300 milliseconds, I wanted to use for each millisecond. Could you please provide me inputs to make it for more accurate player reading time?

Thanks,
Sudhakar

**Comment by gkatsev** [comment]
What do you mean for each millisecond? Like fire an event for 250 and 251? Unfortunately, even js itself doesn't really have timing granularity that fine.
This plugin uses `timeupdate` handlers to know when to hide and show, these run anywhere between 50ms to 250ms depending on the browser.
We could switch to use requestAnimationFrame but using that as a timer will use a lot of power and isn't necessary for most usecases of this plugin.

**Comment by sudha53633** [comment]
Dear @gkatsev Thanks for quick response, Yup i too red regarding videojs. Do we have any chance to fire an event for each millisecond with HTML5 video tag?  Or would you please suggest any other video plugins?

**Comment by gkatsev** [comment]
I don't think it would be possible to fire an event for every millisecond and I'm not sure that any plugin out there will support your usecase. You're probably better of writing it yourself.

**Comment by sudha53633** [comment]
Yup that's better. Thanks for your inputs. But even HTML5 timetoupdate or
any other events should support right?
